client/components/token-field/test/index.jsx

```diff
@@ -361,7 +361,7 @@ describe( 'TokenField', () => {
 			firstSuggestion.simulate( 'click' );
 
 			// wait for setState call
-			jest.runTimersToTime( 10 );
+			jest.advanceTimersByTime( 10 );
```

Comment:
Renamed and deprecated https://jestjs.io/docs/timer-mocks#renamed-from-runtimerstotime-to-advancetimersbytime-in-jest-2200